habitat-lab/habitat/config/benchmark/rearrange/play_spot.yaml

```diff
@@ -23,6 +25,11 @@ habitat:
         disable_grip: False
         delta_pos_limit: 0.0125
         ee_ctrl_lim: 0.015
+      base_velocity_spot:
+        allow_dyn_slide: False
+        cylinder_deviate: 0.3
+        lin_collision_threshold: 0.04
+        ang_collision_threshold: 0.01
```

Comment:
This seems dangerous, especially with continuous control.
Currently we enforce a maximum error of 1e-5 (see [agent action controls](https://github.com/facebookresearch/habitat-sim/blob/8f9cc41a7d87894032da70c95f6b427c7faf113d/src_python/habitat_sim/agent/controls/object_controls.py#L75)).
I feel like we should have the same requirements here. Any planar error is the result of a collision, right?

Reply:
Thank you for pointing out this! I make it to be small!